manifests/p/Phoner/PhonerLite/2.93.0/Phoner.PhonerLite.yaml

```
@@ -0,0 +1,27 @@
+PackageIdentifier: Phoner.PhonerLite
+Publisher: Heiko Sommerfeldt
+PackageName: PhonerLite
+Moniker: PhonerLite
+PackageVersion: 2.93.0
```

```suggestion
PackageVersion: 2.93
```
**You'll have to update the version folder to match `PackageVersion` using GitHub Desktop from https://desktop.github.com on Windows**

This change was made to match the application's version with the one shown in Control Panel.

---

Why version 2.93.0 is not OK? When I delete last ".0" the path is not valid anymore. I don't know how to change both - path and version.

---

If you want the version to be `2.93.0` you'll have to update your application so that it shows up as `2.93.0` rather than `2.93`.

If the version is kept as `2.93.0`, Windows Package Manager will always update the application regardless if you're on the latest version of the application or not.

---

I can do that for you :)

---

It's merged now.